lib/cgroup/cgroup_test.go

```diff
@@ -27,106 +27,100 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 
 	"github.com/google/uuid"
-	"gopkg.in/check.v1"
+	"github.com/stretchr/testify/require"
 )
 
 func TestMain(m *testing.M) {
 	utils.InitLoggerForTests()
 	os.Exit(m.Run())
 }
 
-type Suite struct{}
-
-var _ = check.Suite(&Suite{})
-
-func TestControlGroups(t *testing.T) { check.TestingT(t) }
-
 // TestCreate tests creating and removing cgroups as well as shutting down
 // the service and unmounting the cgroup hierarchy.
-func (s *Suite) TestCreate(c *check.C) {
+func TestCreate(t *testing.T) {
 	// This test must be run as root. Only root can create cgroups.
 	if !isRoot() {
-		c.Skip("Tests for package cgroup can only be run as root.")
+		t.Skip("Tests for package cgroup can only be run as root.")
 	}
 
 	// Create temporary directory where cgroup2 hierarchy will be mounted.
 	dir, err := os.MkdirTemp("", "cgroup-test")
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 	defer os.RemoveAll(dir)
 
 	// Start cgroup service.
 	service, err := New(&Config{
 		MountPath: dir,
 	})
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	// Create fake session ID and cgroup.
 	sessionID := uuid.New().String()
 	err = service.Create(sessionID)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	// Make sure that it exists.
 	cgroupPath := path.Join(service.teleportRoot, sessionID)
 	_, err = os.Stat(cgroupPath)
 	if os.IsNotExist(err) {
-		c.Fatalf("Could not find cgroup file %v: %v.", cgroupPath, err)
+		t.Fatalf("Could not find cgroup file %v: %v.", cgroupPath, err)
```

Comment:
Replace all of this with `require.FileExists(t, cgroupPath)` (or `DirExists` if it's a directory)